sdcm/sct_config.py

```diff
@@ -1593,6 +1596,30 @@ def _check_backend_defaults(backend, required_params):
             'new_scylla_repo', 'scylla_repo_m', 'scylla_repo_loader', 'scylla_mgmt_repo', 'scylla_mgmt_agent_repo',
             'scylla_mgmt_agent_repo'] if self.get(url)))
 
+    def verify_configuration(self):  # pylint: disable=too-many-statements
+        """
+        Check that all required values are set, and validated each value to be of correct type or value
+        also check required options per backend
+        Also check if ami_ids and urls are valid
+
+        :return: None
+        :raises ValueError: on failures in validations
+        :raise Exception: on unsupported backends
+        """
+        self._verify_configuration()
+        self._verify_configuration_urls_validity()
+
+    def verify_configuration_no_url_validation(self):  # pylint: disable=too-many-statements
+        """
+        Check that all required values are set, and validated each value to be of correct type or value
+        also check required options per backend
+
+        :return: None
+        :raises ValueError: on failures in validations
+        :raise Exception: on unsupported backends
+        """
+        self._verify_configuration()
```

Comment:
These wrappers look redundant. Why we cannot just call both method in sct.py directly?
It is completely clear what does "_verify_configuration_urls_validity" do. No need to have wrapper for it.